requirements_plugins.txt

```diff
@@ -0,0 +1,7 @@
+# Plugins:
+plover-treal==1.0.0
+
+# Dependencies:
+hidapi==0.7.99.post20
```

Comment:
I don't think there's another option: the list of requirements can vary — depending on the OS, Python version — when requirement markers are used... We can make it easier to find the list of requirements needed for each dependency however: https://github.com/benoit-pierre/plover/compare/no_treal...benoit-pierre:helper_for_updating_requirements

So to add a new standard plugin to the distributions:
1. create a new paragraph for in `requirements_plugins.txt`, only add a requirement for the plugin itself
2. push to GitHub, and check the output of listing the distribution packages

One issue: we don't build the Linux distribution when not deploying... And the standard plugins are not installed per default; because you can't do it at the same time as the other dependencies since they will depend on Plover... ¯\\_(ツ)_/¯